runtime/gc_vlhgc/AllocationContextBalanced.hpp

```diff
@@ -71,6 +71,9 @@ class MM_AllocationContextBalanced : public MM_AllocationContextTarok
 	UDATA *_freeProcessorNodes;	/**< The array listing all the NUMA node numbers which account for the nodes with processors but no memory plus an empty slot for each context to use (element 0 is used by this context) - this is used when setting affinity */
 	UDATA _freeProcessorNodeCount;	/**< The length, in elements, of the _freeProcessorNodes array (always at least 1 after startup) */
 
+#if defined(J9VM_GC_SPARSE_HEAP_ALLOCATION)
+	MM_HeapRegionDescriptorVLHGC *_leafRegionList; /** for Off-heap case only, adding and removing via region->_allocateData.pushRegionToLeafRegionList/popRegionFromLeafRegionList */
```

Comment:
this is more descriptive of what this really is:
the list of regions reserved and decommited by all large array allocations, instead of which some other contiguous memory (offheap) is actually committed and used by those arrays

Reply:
we could even rename it as: _arrayReservedRegionList (@dmitripivkine opinion?)

we should start retiring terms like arraylet and even leaf, in offheap specific paths